docs/core/extensions/snippets/configuration/custom-provider/Providers/EntityConfigurationSource.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+
+namespace CustomProvider.Example.Providers
+{
+    public class EntityConfigurationSource : IConfigurationSource
+    {
+        readonly Action<DbContextOptionsBuilder> _optionsAction;
```

Comment:
I'd prefer this to be explicitly private:

```suggestion
        private readonly Action<DbContextOptionsBuilder> _optionsAction;
```